app/assets/stylesheets/common/base/modal.scss

```diff
@@ -709,3 +709,80 @@
     width: 10%;
   }
 }
+
+.create-invite-modal {
+  .input-group {
+    margin-bottom: 1em;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+
+    input[type="text"] {
+      width: 100%;
+
+      &.invite-link {
+        width: 85%;
+      }
+    }
+  }
+
+  .radio-group {
+    input[type="radio"] {
+      display: table-cell;
+      vertical-align: middle;
+      margin-top: -1px;
+    }
+
+    label {
+      display: inline-block;
+    }
+  }
+
+  .group-chooser,
+  .future-date-input-selector {
+    width: 100%;
+  }
+
+  .input-group input[type="text"],
```

Comment:
@awesomerobot just pinging you here cause I feel like no one should feel the need to do this